using_images/other_images/jenkins.adoc

```diff
@@ -109,3 +109,14 @@ Once selected, you must link:../../dev_guide/templates.html#creating-resources-f
 == Tutorial
 
 For more details on the sample job included in this image, see this link:https://github.com/openshift/origin/blob/master/examples/jenkins/README.md[tutorial].
+
+ifdef::openshift-origin[]
+== OpenShift Pipeline Plugin
+
+The Jenkins image's list of pre-installed plugins now includes a plugin which assists in the creating of CI/CD workflows that run against
+an OpenShift server.  A series of build steps, post-build actions, as well as SCM-style polling are provided which equate to administrative
+and operational actions on the OpenShift server and the API artifacts hosted there.
+
+See the https://github.com/openshift/jenkins-plugin/[the plugin's README] for a detailed description of what is available.
```

Comment:
make some comment about the fact that the sample job that is pre-configured in the image also utilizes this plugin and can serve as a usage example.